src/pages/tutorials/smart-rollups/optimize.md

```diff
@@ -0,0 +1,68 @@
+---
+id: optimize
+title: "Part 4: Optimizing the kernel"
+lastUpdated: 11th October 2023
+---
+
+The kernel must be efficient with space and processing power because there is a size limit for kernels.
```

Comment:
The `processing power` thing is kind of unrelated to the size of the kernel. In general, there's also not a size limit for kernels (or rather, it's stupidly big) - it's only for origination where this matters.

I'm not sure `optimize` is the correct word here I guess, to a developer, it sounds like we're going to be making it faster, but actually we're just working around the limit of origination size.

Reply:
I can't come up with a better word than "optimize." "Compress?" "Reduce the size on disk of?"

Changed this line to:
> To originate the kernel on Tezos, it must fit within the maximum size for a layer 1 operation.